src/Core/Components/Dialog/FluentDialog.razor.css

```diff
@@ -3,12 +3,13 @@
     position: relative;
     padding: 0px var(--dialog-padding);
     width: calc(var(--dialog-width) - 2 * var(--dialog-padding));
-    height: calc(var(--dialog-height) - 2 * var(--dialog-padding));
-    max-width: 80%;
 }
 
-fluent-dialog:has(div[class~="fluent-dialog-body"]):not(:is(fluent-dialog[class~="right"], fluent-dialog[class~="left"]) )::part(control) {
+fluent-dialog.autoheight::part(control) {
```

Comment:
When `Height` is a specified value. How the height is set, if you removed `height: calc(var(--dialog-height) - 2 * var(--dialog-padding));`?

Is it possible to keep the height in `fluent-dialog::part(control)` and to override with this style and an `!important` flag?

Reply:
Height is then set as `--dialog-height` in the style of the fluent-dialog. There is actually no need to set the heigth in the  part(control) again

Reply:
Including the `2 * var(--dialog-padding)`? So, why to keep `width`?